js/components/dateRange.js

```diff
@@ -46,37 +46,34 @@ Fliplet.FormBuilder.field('dateRange', {
       isInputFocused: false,
       isPreview: Fliplet.Env.get('preview'),
       today: this.formatDate(),
-      selectedRange: {
-        label: T('widgets.form.dateRange.rangePlaceholder'),
-        value: ''
-      },
+      selectedRange: null,
       predefinedRanges: [
         {
-          label: T('widgets.form.dateRange.predefinedRanges.today'),
+          label: 'Today',
```

Comment:
The labels for predefined ranges need to be translated because these labels are not provided by the user